l5kit/l5kit/dataset/ego.py

```diff
@@ -1,18 +1,25 @@
 import bisect
+import warnings
 from functools import partial
+from pathlib import Path
 from typing import Callable, Optional
 
 import numpy as np
 from torch.utils.data import Dataset
+from zarr import convenience
 
-from l5kit.data import ChunkedDataset, get_frames_slice_from_scenes
+from l5kit.data import ChunkedDataset, get_agents_slice_from_frames, get_frames_slice_from_scenes
+from l5kit.data.labels import PERCEPTION_LABEL_TO_INDEX
 from l5kit.dataset.utils import convert_str_to_fixed_length_tensor
 from l5kit.kinematic import Perturbation
 from l5kit.rasterization import Rasterizer, RenderContext
 from l5kit.sampling.agent_sampling import generate_agent_sample
 from l5kit.sampling.agent_sampling_vectorized import generate_agent_sample_vectorized
 from l5kit.vectorization.vectorizer import Vectorizer
 
+from .select_agents import select_agents, TH_DISTANCE_AV, TH_EXTENT_RATIO, TH_YAW_DEGREE
```

Comment:
nit: given there are a few constants, I would just do a `import .select_agents as sa` or something and then use it prefixed like `sa.TH_DISTANCE_AV`, just to avoid adding importing it directly in the namespace.